### DISCUSSION TITLE
33.1.1.176 eligible for release?

### DISCUSSION THREAD

**Discussion by marcanw** [airsdk/Adobe-Runtime-Support#598] [Q&A]
Just asking because it has been removed from the download page: https://airsdk.harman.com/download
I know there was few page edition issues in the past but I'm always nervous before a release...

**Comment by ajwfrost** [comment]
Eligible for release, yes. But it was causing enough problems that we rolled back to .98 on the main download page; we're aiming to get another release out that fixes some of the (mostly Mac-based) problems at the end of this week..

**Comment by ajwfrost** [comment]
(To clarify 'problems' are mostly toolchain ones i.e. in the SDK itself - primarily the Mac-based iOS development and deployment)

**Comment by dmunsie** [comment]
Can you please have links to older sdks? Some users may have a need to roll back to an earlier sdk, especially in dire situations where bugs are found in the current release days, weeks after it's released. Thank you for your consideration.

**Comment by ajwfrost** [comment]
Yes, this is in progress, we should be able to start getting these links up within the next week or two I believe.

We’d been planning to just start with the 33.1.1.98 release though, I’m not sure it’s worth going any further back! - and then all future releases will be available probably via the release notes page.

Thanks
